mesonbuild/environment.py

```diff
@@ -913,6 +913,9 @@ def get_build_dir(self) -> str:
 
     def get_import_lib_dir(self) -> str:
         "Install dir for the import library (library used for linking)"
+        m = self.machines.host
+        if m.is_haiku():
+            return 'develop/lib'
```

Comment:
On windows we know that we can just use the bindir for certain types of file, which is part of the user configuration too.

For Haiku, this is hardcoding a location that only seems to make sense if you know the default libdir hasn't been modified. It's totally opaque to GNU Directory Variables etc. This seems very confusing.

Reply:
Should `developLibDir` option be introduced or what is preferred solution?